tests/FSharp.Compiler.Service.Tests/CSharp_Analysis/CSharp_Analysis.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard20</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
I think I've seen this change about 3 times now :)

Reply:
lol

Reply:
If I keep commenting, will you keep approving?